WcaOnRails/app/models/competition.rb

```diff
@@ -233,6 +234,7 @@ def update_foreign_keys
       CompetitionDelegate,
       CompetitionOrganizer,
       DelegateReport,
+      CompetitionTab,
     ].each do |model|
       foreign_key = model.column_names.include?("competitionId") ? "competitionId" : "competition_id"
       model.where(foreign_key => id_was).update_all(foreign_key => id)
```

Comment:
That's a different approach than what I would have done, but it should work quite well. The vulnerability I can see is that if you don't set up the relationship in the `Competition` class, that table will be missed.